tools/serverpod_cli/test/analyzer/entities/entity_analyzer/field_scope_test.dart

```diff
@@ -198,6 +214,14 @@ void main() {
           [definition],
         );
 
+        test('then a deprecated info is collected.', () {
```

Comment:
Suggestion: Validate the log level of the message.

Since we definitely don't want this to become an error for our users it might be good to validate that this is logged as an info error.